ftw/simplelayout/restapi/services.py

```diff
@@ -0,0 +1,7 @@
+from plone.restapi.services import Service
+from ftw.simplelayout.configuration import synchronize_page_config_with_blocks
+
+
+class SynchronizePageConfigWithBlocks(Service):
+    def reply(self):
+        return synchronize_page_config_with_blocks(self.context)
```

Comment:
Did you verify if you can return a python data structure or if you need to return a `json` string?

Reply:
Yes. It is possible to just return a dict, the plone.restapi dies the rest😏